hail/src/main/scala/is/hail/expr/ir/agg/Extract.scala

```diff
@@ -391,55 +400,41 @@ class Aggs(
     ResultOp.makeTuple(aggs)
 }
 
+// --- Extract --
+
```

Comment:
Were you going to complete this comment?

Reply:
I was trying to impose some logical sections in the file, partly for my own understanding. But I think to be effective the section dividers would have to be much bigger, and preferably have some top level comments explaining the section. I'll remove for now.